src/migrations/r-migration-commands/m131_132_billboarding_to_xy_default.ts

```diff
@@ -0,0 +1,33 @@
+import { RecordNode, r, rtp, RT, en } from "../../r";
+import { IOrder } from "../IOrder";
+import { BillboardingTypes } from "../../index";
+
+/** 
+ * Update billboarding from boolean to string.
+ */
+
+class Migration implements IOrder {
+  execute (projectJson: unknown) {
+    const pJson = projectJson as RecordNode<RT.project>;
+    const projectF = r.project(pJson);
+
+    const billboardingElements = projectF.getAllDeepChildrenWithFilter(RT.element, el => el.props.billboarding !== undefined);
+    
+    for (const el of billboardingElements) {
+      const elementF = r.element(el);
+      const currentBillboarding = elementF.getValueOrDefault(rtp.element.billboarding);
+      if (currentBillboarding === true) {
+        elementF.set(rtp.element.billboarding, BillboardingTypes.XY);
```

Comment:
Need this to be XYZ for exisitng projects to work as they already do. Current impl is XYZ